src/Microsoft.DotNet.Build.Tasks.Feed/BlobFeedAction.cs

```diff
@@ -2,16 +2,17 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using Microsoft.Build.Framework;
+using Microsoft.WindowsAzure.Storage;
```

Comment:
This was an initial concern I had especially in the context of build from source, but we can just opt out of building this from the build.

Reply:
Sleet depends on it. What was the issue with it?

Reply:
Nothing, just wanted to confirm that this is a new dependency being included.

Reply:
What library/package are these API's in? Do we already pull them into the dependency list?

Reply:
@weshaggard  Yes, Microsoft.Build.Framework is part of the Microsoft.Build.Framework package and Microsoft.WindowsAzure.Storage is contained in Sleet which is part of the dependency list

Reply:
>Microsoft.WindowsAzure.Storage is contained in Sleet which is part of the dependency list

By contained I assume you mean a dependency? Hopefully that dependency works on dotnet core.